package.json

```diff
@@ -5,12 +5,14 @@
   "dependencies": {
     "ajv": "6.5.4",
     "chart.js": "2.7.2",
+    "highcharts": "^6.2.0",
     "lodash": "4.17.10",
     "moment": "2.22.1",
     "react": "16.3.2",
     "react-chartjs-2": "2.7.2",
     "react-dom": "16.3.2",
     "react-helmet": "5.2.0",
+    "react-jsx-highcharts": "^3.3.1-alpha.1",
```

Comment:
Not sure why `npm install...` pulled in an alpha version. Not sure what to do about that either.

Reply:
@todo